vrecord.rb

```diff
@@ -8,7 +8,7 @@ class Vrecord < Formula
 
   bottle :unneeded
```

Comment:
FYI: This is no longer useful. I will remove from all formulae in a PR next weekend.